pymove/utils/data_augmentation.py

```diff
@@ -368,3 +373,119 @@ def instance_crossover_augmentation(
         traj_df, restriction=restriction, frac=frac
     )
     insert_points_in_df(data, aug_df)
+
+
+def sliding_window(
+    data: DataFrame,
+    size_window: Optional[int] = 6,
+    size_jump: Optional[int] = 3,
+    columns: Optional[List] = None,
+) -> DataFrame:
+    """
+    Sliding window technique.
+
+    Performs an increase in the trajectory data by sliding a window
+    Over each sequence to a specified size n, skipping m points.
+    This process inserts sub-trajectories in the data set.
+
+    Parameters
+    ----------
+    data: pandas.core.frame.DataFrame
+        Trajectory data in sequence format.
+    size_window: number, optional, default 6
+        Sliding window size.
+    size_jump: number, optional, default 3
+        Size of the jump in the trajectory.
+    columns: list, optional, default None
+        Columns to which the split will be applied.
+
+    Return
+    ------
+    DataFrame
+        Increased data set.
+    """
+    if columns is None:
+        columns = data.columns
+
+    frames = {}
+    desc = 'Sliding Window'
+    for idx, row in progress_bar(data.iterrows(), desc=desc, total=data.shape[0]):
+        frames[idx] = pd.DataFrame(
+            [serie for serie in split_trajectory(
+                row, size_window, size_jump, columns
+            )]
+        )
+
+    data_ = pd.concat(
+        [frames[i] for i in range(len(frames))],
+        ignore_index=True
+    )
+
+    return data_
+
+
+def transition_graph_augmentation_all_vertex(
```

Comment:
Can the operation have the option to be not `inplace`? Also, could the `graph` not be returned?